Resources/Regression/qml/RegressionLogistic.qml

```diff
@@ -107,16 +107,16 @@ Form
                 }
 
             }
-			CheckBox { name: "stdCoeff";		label: qsTr("Standardized coefficients")	}
-			CheckBox { name: "oddsRatios";		label: qsTr("Odds ratios")					}
+			CheckBox { name: "stdCoeff";		label: qsTr("Standardized coefficients"); enabled: coeffEstimates.checked	}
```

Comment:
Shouldn't all of these just be children of `coeffEstimates`? So we're inline with other analyses

Reply:
Hm, yeah you are right, although we will not be really consistent in across analyses regardless what I do here (e.g., confidence interval in linear regression is also not a child of estimates checkbox, but it also does not get enabled/disables depending on the value of it). 

I also find it more visually pleasing as it is now compared to how it looks when I make them children of the estimates button (see screenshots). For example, it seems to me logical then to remove the embedding of these buttons under the “Regression coefficients” group and rename the button “Estimates” to “Regression coefficients”, but then again that would totally go against the convention in other analyses. 

I don’t really mind any of the solutions, but I am not sure what is the best course of action. Any thoughts?

Current status
![Screen Shot 2020-03-23 at 13 02 08](https://user-images.githubusercontent.com/7093546/77314754-8c578a00-6d06-11ea-8ed1-701802e87786.png)

Proposed as children
![Screen Shot 2020-03-23 at 13 02 45](https://user-images.githubusercontent.com/7093546/77314786-9da09680-6d06-11ea-9613-eb94de143ea9.png)

Reply:
I agree that we should use the `QML` syntax to specify them as children. That's nicer and less error-prone than repeating the id many times. However, if everything is indented then nothing is. So that looks a bit redundant. Perhaps we can add a `QML` property to enable/ disable the indenting of children?

Reply:
> Hm, yeah you are right, although we will not be really consistent in across analyses regardless what I do here (e.g., confidence interval in linear regression is also not a child of estimates checkbox, but it also does not get enabled/disables depending on the value of it).

I think it should also be indented in Linear Regression to be honest.

> I also find it more visually pleasing as it is now compared to how it looks when I make them children of the estimates button (see screenshots). For example, it seems to me logical then to remove the embedding of these buttons under the “Regression coefficients” group and rename the button “Estimates” to “Regression coefficients”, but then again that would totally go against the convention in other analyses.

The group is probably called "Regression Coefficients" because that's where they go in Linear Regression as well, only in Logistic Regression it's the only option available for it. So it's a decision between across-analysis-consistency vs within-analysis-clarity I guess...

> Proposed as children

To my eyes it becomes immediately clear that those checkboxes are meant to tweak estimates further. So I think that's actually really nice. Just like Descriptives, it just isn't as clear when it's not indented:
![descriptives_indents](https://user-images.githubusercontent.com/18737241/77406291-b5365880-6db4-11ea-9b7d-d5b9a1357d90.png)

Clarity should be key in my opinion.

Reply:
Ok, ok! You convinced me.